### ISSUE TITLE
bug: fix tfsec violations in module

### ISSUE THREAD

**Issue by reegnz** [lacework/terraform-aws-eks-audit-log#14]
**Describe the bug**
I'm running [tfsec](https://github.com/aquasecurity/tfsec) on my terraform code. When consuming this module I had to add several tfsec ignores, like:

```tf
#tfsec:ignore:aws-s3-block-public-acls
#tfsec:ignore:aws-s3-block-public-policy
#tfsec:ignore:aws-s3-ignore-public-acls
#tfsec:ignore:aws-s3-no-public-buckets
#tfsec:ignore:aws-iam-no-policy-wildcards
#tfsec:ignore:aws-s3-enable-bucket-logging
#tfsec:ignore:aws-s3-specify-public-access-block
module "aws_eks_audit_log" {
  count = var.enable_eks_audit_logs ? 1 : 0

  source  = "lacework/eks-audit-log/aws"
  version = "0.3.0"

  ...
}
```
IMHO this module should follow security best practices for the S3 bucket, and block all public access. Not having logging, or having policy wildcards are more agruable and I wouldn't find it a necessity.

**Steps to reproduce**
Use the module, run `tfsec .` in the folder.

**Expected behavior**
The module shouldn't throw tfsec errors. Either put tfsec ignores in your module, or fix the findings of tfsec.

**Screenshots**

**Please complete the following information):**
 - Terraform Version: v1.2.6
 - Module Version v0.3.0

**Additional context**
I'm integrating our kubernetes clusters with lacework, but I also have to comply with security scanning our terraform codebase.

**Comment by jon-stewart** [comment]
Hi @reegnz, thanks for raising this bug.  I apologise that we are only looking at this now.

**Comment by jon-stewart** [comment]
Hi @reegnz, this should be addressed in the `v0.4.3` release.

Note that there are a couple of tsec ignores in `main.tf`:
1. aws-s3-enable-bucket-logging - tfsec only recognises the deprecated nested approach
2. aws-iam-no-policy-wildcards - tfsec doesn't accept the S3 bucket resource wildcard which is required to make use of the bucket

Thank you again for spotting this.
